### DISCUSSION TITLE
Is madanalysis5 installed within madgraph still usable with delphes output root files?

### DISCUSSION THREAD

**Discussion by rmahbubani** [MadAnalysis/madanalysis5#265] [Q&A]
I ask because the new MG5_aMC@NLO manual doesn't include madanalysis as one of its compatible third-party software, and I am unable to import .root files made with madevent+delphes into madanalysis any more.  When I start madanalysis in reco mode and type display main, I see this: "File extension NOT readable in this session: .lhe .hep .hepmc **.root** .lhe.gz .hep.gz .hepmc.gz .root.gz .lhe.fifo .hep.fifo .hepmc.fifo .root.fifo .lhe.gz.fifo .hep.gz.fifo .hepmc.gz.fifo .root.gz.fifo".  Thanks in advance, Rakhi

**Comment by BFuks** [comment]
Hi Rakhi,

That's odd... MA5 should be compatible with MG5aMC (@oliviermattelaer?).  Can you please try to use MadAnalysis to generate the root file instead (from the HepMC file using Delphes)? Thanks.

Cheers,

Benjamin

**Reply by rmahbubani** [reply]
As far as I can see that requires me to reinstall delphes within a standalone version of Madanalysis with the install command, is that right @BFuks?  Or can I simply point madanalysis to my existing delphes installation within madgraph?  And is there still some subtlety re: delphes isolation?

**Comment by BFuks** [comment]
Here, it is needed to install delphes locally (as MA5 modifies a few things for the interfacing). Those modifications are needed for backward compatibility.

**Reply by rmahbubani** [reply]
Ok will try that first thing tomorrow.

**Reply by rmahbubani** [reply]
Hi @BFuks, generating the root file using a locally installed dephes works fine.  Any clues as to why it doesn't work within the madgraph interface?

**Comment by BFuks** [comment]
I have no idea…

Cheers,

Benj

**Comment by oliviermattelaer** [comment]
Hi,

> I ask because the new MG5_aMC@NLO manual doesn't include madanalysis as one of its compatible third-party software, 

What do you mean here, not sure which document you are refering to. 
In any case, all version of MG5_aMC@NLO have support for madanalysis5

**Reply by rmahbubani** [reply]
Well importing delphes output root files into madanalysis within the madgraph environment no longer works for me, or for two other collaborators, whereas it works with standalone madanalysis (see above)
